angular-mentoring/src/app/main/i-course.ts

```diff
@@ -0,0 +1,7 @@
+export interface ICourse {
```

Comment:
In most cases, it doesn't make sense to put interfaces in the separate file. Consider moving it to the class

Reply:
Renamed all interfaces without prefix "I"